src/main/java/seedu/address/model/Model.java

```diff
@@ -86,6 +86,9 @@ public interface Model {
      */
     void updateFilteredPersonList(Predicate<Person> predicate);
 
+    ObservableList<Item> getFilteredItemList();
+    void updateFilteredItemList(Predicate<Item> predicate);
```

Comment:
Not sure if we need this. The filteredpersonlist is to support the find feature but we don't have that for our catalogue